mirgecom/boundary.py

```diff
@@ -1188,26 +1203,38 @@ def farfield_state(self, dcoll, dd_bdry, gas_model, state_minus, **kwargs):
             state_minus.dim, mass=free_stream_density,
             energy=free_stream_total_energy,
             momentum=free_stream_density*free_stream_velocity,
-            species_mass=free_stream_spec_mass
-        )
+            species_mass=free_stream_spec_mass)
 
         return make_fluid_state(cv=cv_infinity, gas_model=gas_model,
                                 temperature_seed=free_stream_temperature,
                                 smoothness_mu=state_minus.smoothness_mu,
                                 smoothness_kappa=state_minus.smoothness_kappa,
                                 smoothness_beta=state_minus.smoothness_beta)
 
+    def state_bc(self, dcoll, dd_bdry, gas_model, state_minus, **kwargs):
+        """Return BC fluid state."""
+        return self.state_plus(dcoll, dd_bdry, gas_model, state_minus, **kwargs)
+
```

Comment:
Yeah, I've being thinking about this. Since this is a far-field BC, then the values should converge. But I don't see why not use an average. What are your thoughts?

Reply:
(As discussed in Slack, this appears to be what Mengaldo 2014 prescribes for this case.)